webservices/args.py

```diff
@@ -226,6 +226,15 @@ def make_seek_args(field=fields.Int, description=None):
     'beginning_image_number': fields.List(fields.Str, description=docs.BEGINNING_IMAGE_NUMBER),
     'report_type': fields.List(fields.Str, description='Report type; prefix with "-" to exclude'),
     'is_amended': fields.Bool(description='Report has been amended'),
+    'committee_ids': fields.List(fields.Str, description=docs.COMMITTEE_ID),
+    'min_disbursements_amount': Currency(description='Filter for all amounts greater than a value.'),
+    'max_disbursements_amount': Currency(description='Filter for all amounts less than a value.'),
+    'min_receipts_amount': Currency(description='Filter for all amounts greater than a value.'),
+    'max_receipts_amount': Currency(description='Filter for all amounts less than a value.'),
+    'min_cash_on_hand_amount': Currency(description='Filter for all amounts greater than a value.'),
+    'max_cash_on_hand_amount': Currency(description='Filter for all amounts less than a value.'),
+    'type': fields.List(fields.Str, description=docs.COMMITTEE_TYPE),
+    'auth_committee_candidate_id': fields.List(fields.Str, description=docs.COMMITTEE_ID),
```

Comment:
As for why there is a separate `committee_ids` argument, I think you stated the reason.  This resource serves up two endpoints, and one of those endpoints is `reports\{committee_id}`.  I originally did just try using the committee_id and removed the committee_id path parameter, this of course caused the tests to fail.

So the approach to that is to either route `report\{committee_id}` to a different endpoint than this one, or use a separate `committee_ids` query param, as I did here.  Unless there is some way to do both with Flask, which didn't seem possible to me, but maybe there is a way?